### ISSUE TITLE
Purpose of handler

### ISSUE THREAD

**Issue by leonelgalan** [zzet/ansible-rbenv-role#27]
What's the purpose of `rehash rbenv`? Rehash is called manually from [tasks/main.yml:151](https://github.com/zzet/ansible-rbenv-role/blob/e10d6f8ec5177c1800b9e6753458a667bf222916/tasks/main.yml#L151) or [tasks/main.yml:182](https://github.com/zzet/ansible-rbenv-role/blob/e10d6f8ec5177c1800b9e6753458a667bf222916/tasks/main.yml#L182)

It it notifies the handler instead, we could install gems (`env: system` so now default gems plugin available) and `rbenv rehash` would happen at the very end.

**Comment by zzet** [comment]
@leonelgalan this issue is actual now?

**Comment by leonelgalan** [comment]
Yeah, but I discovered that handlers are only "triggered" at the end of the playbook's run. So if we install bundler, we need to manually `rbenv rehash` in order to have bundle available before the playbook finishes.

I wish there was a handler-like task that could be triggered at the end of the recipe's run or manually for DRYness. So it's might be a non-issue.

**Comment by zzet** [comment]
I have nothing against to run `rehash` immediately after installation `bundler`. I think it is a true way.

**Comment by zzet** [comment]
If you think that should be divided `rbenv global {{rbenv.ruby_version}}` & `rbenv renash` - I think that this can be done. Essentially it will not change, but it will be obvious.
